hw9_task_1.py

```diff
@@ -0,0 +1,87 @@
+class Business():
+
+    def __init__(self, title=None, foundation_year=None, specialize=None, regional_offices=None,
+                 ceo=None):
+        self.__title = title
+        self.__foundation_year = foundation_year
+        self.__specialize = specialize
+        self.__regional_offices = regional_offices
+        self.__ceo = ceo
+
+    @property
+    def title(self):
+        return self.__title
+
+    @property
+    def foundation_year(self):
+        return self.__foundation_year
+
+    @property
+    def specialize(self):
+        return self.__specialize
+
+    @property
+    def regional_offices(self):
+        return self.__regional_offices
+
+    @property
+    def ceo(self):
+        return self.__ceo
+
+    @title.setter
+    def title(self, new_title):
+        if isinstance(new_title, str):
+            self.__title = new_title
+        else:
+            raise TypeError
+
+    @foundation_year.setter
+    def foundation_year(self, new_foundation_year):
+        if isinstance(new_foundation_year, int) and len(str(new_foundation_year)) == 4:
+            self.__foundation_year = new_foundation_year
```

Comment:
looks like I can change the foundation year to the future. 
for example, I can set the year 9999